cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/transaction/stream/StreamAdminTest.java

```diff
@@ -58,13 +78,30 @@
 
   protected abstract InMemoryAuditPublisher getInMemoryAuditPublisher();
 
+  protected abstract Authorizer getAuthorizer();
```

Comment:
does this need to be abstract? Both the subclasses use the same implementation right? Or is this because the injectors are defined in the subclasses? If so, does it make sense to instead declare the injector in the base class and initialize in the subclasses?